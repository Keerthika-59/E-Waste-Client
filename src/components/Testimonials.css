body {
    font-family: 'Roboto Condensed', sans-serif;
}

.testimonial {
    background: #fff;
    text-align: center;
    padding: 30px 30px 50px;
    margin: 0 15px 100px;
    position: relative;
    margin-top: 168px;
}

.testimonial:before,
.testimonial:after {
    content: "";
    border-top: 40px solid #fff;
    border-right: 125px solid transparent;
    position: absolute;
    bottom: -40px;
    left: 0;
}

.testimonial:after {
    border-right: none;
    border-left: 125px solid transparent;
    left: auto;
    right: 0;
}

.icon {
    display: inline-block;
    font-size: 80px;
    color: #bd986b;
    margin-bottom: 20px;
    opacity: 0.6;
}

.description {
    font-size: 16px;
    color: #777;
    text-align: justify;
    margin-bottom: 30px;
    opacity: 0.9;
    letter-spacing: -1px;
}

.testimonial-content {
    width: 100%;
    position: absolute;
    left: 0;
}

.pic {
    display: inline-block;
    border: 6px solid white;
    border-radius: 50%;
    box-shadow: 0 0 2px 2px #daad86;
    overflow: hidden;
    z-index: 1;
    position: relative;
}

.image {
    width: 100%;
    height: auto;
}

.title {
    font-size: 15px;
    font-weight: bold;
    color: #fff;
    text-transform: capitalize;
    margin: 0 0 5px 0;
}

.post {
    display: block;
    font-size: 14px;
    color: #ffd9b8;
}

.owl-controls {
    margin-top: 60px;
}
.owl-page span {
    width: 32px;
    height: 10px;
    background: #fff;
    border: 2px solid #bd986b;
    margin: 5px;
    opacity: 1;
}
.owl-page.active span,
.owl-page:hover span {
    background: #ffd9b8;
    border-color: #fff;
}

img {
    width: 100px !important;
    height: 50px;
}