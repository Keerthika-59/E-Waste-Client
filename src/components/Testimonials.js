import React from 'react'

 function Testimonials(){
    return(
        <>
        <div className="demo" style={{background: `url('${process.env.PUBLIC_URL}/assets/images/background.jpg')`}}>
        <div className="container">
        <div className="row">
            <div className="col-md-12">
                <div id="testimonial-slider" className="owl-carousel">
                    <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="images.jpg" className="image" alt=""/></div>
                            <h3 className="title">williamson</h3>
                            <span className="post">Web Developer</span>
                        </div>
                    </div>
                    <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="3.jpg" className="image" alt=""/></div>
                            <h3 className="title">Kristina</h3>
                            <span className="post">Web Designer</span>
                        </div>
                    </div>
                    <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="team-3.jpg" className="image" alt=""/></div>
                            <h3 className="title">Steve Thomas</h3>
                            <span className="post">Web Developer</span>
                        </div>
                    </div>
                       
                       <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="team-6.jpg" className="image" alt=""/></div>
                            <h3 className="title">Steve Thomas</h3>
                            <span className="post">Web Developer</span>
                        </div>
                    </div>     
                       <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="team-4.jpg" className="image" alt=""/></div>
                            <h3 className="title">Steve Thomas</h3>
                            <span className="post">Web Developer</span>
                        </div>
                    </div>     
                       <div className="testimonial">
                        <span className="icon fa fa-quote-left"></span>
                        <p className="description">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam justo neque, aliquet sit amet elementum vel, vehicula eget eros. Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.Vivamus arcu metus, mattis sed sagittis at, sagittis quis neque. Praesent.
                        </p>
                        <div className="testimonial-content">
                            <div className="pic"><img src="team-6.jpg" className="image" alt=""/></div>
                            <h3 className="title">Steve Thomas</h3>
                            <span className="post">Web Developer</span>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
        </>
    )
 }

 export default Testimonials;