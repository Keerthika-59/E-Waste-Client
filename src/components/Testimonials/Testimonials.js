import React from "react";
import { Carousel, Container, Card } from 'react-bootstrap';
import './Testimonials.css';
const Testimonials = () => {

    return (
        <Container fluid={true}>
            <Carousel className="testimonial">
                <Carousel.Item interval={1000}>
                    <Card className="test1">
                        <div class="testcontainer">
                            <div class="testcarousel-caption">
                                <p>If Shai Reznik's TDD videos don't convince you to add automated testing your code, I don't know what will.This was the very best explanation of frameworks for brginners that I've ever seen. </p> <img src="https://i.imgur.com/lE89Aey.jpg" alt="" />
                                <div id="image-caption">Nick Doe</div>
                            </div>
                        </div>
                    </Card>
                </Carousel.Item>
                <Carousel.Item >
                    <Card className="test1">
                        <div class="testcontainer">
                            <div class="testcarousel-caption">
                                <p>If Shai Reznik's TDD videos don't convince you to add automated testing your code, I don't know what will.This was the very best explanation of frameworks for brginners that I've ever seen. </p> <img src="https://i.imgur.com/lE89Aey.jpg" alt="" />
                                <div id="image-caption">Nick Doe</div>
                            </div>
                        </div>
                    </Card>
                </Carousel.Item>
                <Carousel.Item >
                    <Card className="test1">
                        <div class="testcontainer">
                            <div class="testcarousel-caption">
                                <p>If Shai Reznik's TDD videos don't convince you to add automated testing your code, I don't know what will.This was the very best explanation of frameworks for brginners that I've ever seen. </p> <img src="https://i.imgur.com/lE89Aey.jpg" alt="" />
                                <div id="image-caption">Nick Doe</div>
                            </div>
                        </div>
                    </Card>
                </Carousel.Item>
            </Carousel>
        </Container>
    );
};

export default Testimonials;