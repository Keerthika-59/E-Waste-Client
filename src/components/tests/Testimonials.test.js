import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
import Testimonials from '../Testimonials/Testimonials';

test('renders testimonials', () => {
    const div=document.createElement("div");
    <Route>
        ReactDom.render(<Testimonials/>,div);
    </Route>
    ReactDOM.unmountComponentAtNode(div);
   });
   test('should test Testimonials component', () => {
    const wrapper = shallow(<Testimonials />);
    expect(wrapper).toMatchSnapshot();
   });