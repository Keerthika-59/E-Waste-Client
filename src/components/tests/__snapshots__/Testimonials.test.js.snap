// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Testimonials component 1`] = `
<Container
  fluid={true}
>
  <Carousel
    className="testimonial"
    controls={true}
    defaultActiveIndex={0}
    fade={false}
    indicators={true}
    interval={5000}
    keyboard={true}
    nextIcon={
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
    }
    nextLabel="Next"
    pause="hover"
    prevIcon={
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
    }
    prevLabel="Previous"
    slide={true}
    touch={true}
    wrap={true}
  >
    <CarouselItem
      interval={1000}
    >
      <Card
        body={false}
        className="test1"
      >
        <div
          class="testcontainer"
        >
          <div
            class="testcarousel-caption"
          >
            <p>
              The need of the handling the waste the right way is more important now than never, the waste needs to be properly disposed to prevent further harm to the environment. 
            </p>
             
            <img
              alt=""
              src="assets/images/img3.jpg"
              style={
                Object {
                  "height": "100px",
                }
              }
            />
            <div
              id="image-caption"
            >
              Nick Doe
            </div>
          </div>
        </div>
      </Card>
    </CarouselItem>
    <CarouselItem>
      <Card
        body={false}
        className="test1"
      >
        <div
          class="testcontainer"
        >
          <div
            class="testcarousel-caption"
          >
            <p>
               We really need to step up to change the way the waste being handled for our younger generation. And i can say that using this application we are one step close to. 
            </p>
             
            <img
              alt=""
              src="assets/images/img4.jpg"
              style={
                Object {
                  "height": "100px",
                }
              }
            />
            <div
              id="image-caption"
            >
              Robert
            </div>
          </div>
        </div>
      </Card>
    </CarouselItem>
    <CarouselItem>
      <Card
        body={false}
        className="test1"
      >
        <div
          class="testcontainer"
        >
          <div
            class="testcarousel-caption"
          >
            <p>
              I am happy that the items that i no longer use or need are able to reach the hands of the people who need them the most giving me an opportunity to serve the society. 
            </p>
             
            <img
              alt=""
              src="assets/images/img1.jpeg"
              style={
                Object {
                  "height": "100px",
                }
              }
            />
            <div
              id="image-caption"
            >
              Chris
            </div>
          </div>
        </div>
      </Card>
    </CarouselItem>
  </Carousel>
</Container>
`;
